server/prisma/seed.ts

```diff
@@ -0,0 +1,68 @@
+import { PrismaClient } from '@prisma/client';
+import { Node } from '@prisma/client';
+
+const prisma = new PrismaClient(); 
+async function main() {
+  await prisma.$transaction(async (tx) => {
+    const user = await tx.user.create({
+      data: {
+        username: 'admin',
+        email: 'admin@admin.net',
+        password: '$2a$10$EhUTo57qlj7sy6BlbugnnuUedOJ4HRqLiyYXP8wT46lz3ICKk.k/e'
+      }
+    });
+        
+    const mindmap = await tx.mindmap.create({
+      data: {
+        name: 'Mindmap Seed 1',
+        userId: user.id
+      }
+    });
+
+    const nodes: Node[] = [];
+        
+    for (let i = 0; i < 10; i++) {
+      const node = await tx.node.create({
+        data: {
+          label: 'Node ' + i.toString(),
+          xPos: Math.random(),
+          yPos: Math.random(),
+          color: '#000000',
+          shape: 'circle',
+          mindmapId: mindmap.id
+        }
+      });
+      nodes.push(
+        node
+      );
+    }
```

Comment:
The position of a node can possibly be improved by setting, for example, the coordinates of a single location for the creation of each new node